get_google_oauth.js

```diff
@@ -3,15 +3,14 @@ const readline = require('readline');
 const {google} = require('googleapis');
 
 const SCOPES = [
-    'https://www.googleapis.com/auth/admin.directory.group.member'
-    ,'https://www.googleapis.com/auth/admin.directory.group'
-    ,'https://mail.google.com/'
-    ,'https://www.googleapis.com/auth/gmail.modify'
-    ,'https://www.googleapis.com/auth/gmail.compose'
-    ,'https://www.googleapis.com/auth/gmail.send'
+  'https://www.googleapis.com/auth/admin.directory.group.member',
+  'https://www.googleapis.com/auth/admin.directory.group',
+  'https://mail.google.com/',
+  'https://www.googleapis.com/auth/gmail.modify',
+  'https://www.googleapis.com/auth/gmail.compose',
+  'https://www.googleapis.com/auth/gmail.send',
```

Comment:
This change is fine, but FYI, my original reason for leading commas like this is because adding items to the end of a list is more common than the beginning. In languages that don't allow a final comma at the end, this leading comma style helps prevent syntax errors when adding new elements. I do think the style is ugly, but it helps prevent problems.

Then I found out JavaScript added final commas to everything between ES5 and ES2017, so this ugly style isn't necessary.

Reply:
Yeah, trailing commas alleviates the benefits of some styles that focus on diff-cleanliness.